docs/features/dead_letter_archive.md

```diff
@@ -19,3 +19,80 @@ In addition to the above, as of Cumulus v9+, the Cumulus API also contains a new
+    md5OfBody: [string], // checksum of message body
+    eventSource: [string], // aws:sqs
+    awsRegion: [string], // aws region that this is happening in
+    messageId: [string], // uniqueID of the DLQ message
+    receiptHandle: [string], // An identifier associated with the act of receiving the message. A new receipt handle is returned every time you receive a message.
+    attributes: [Object], // A map of the attributes requested in ReceiveMessage to their respective values.
+    messageAttributes: [Object], // Each message attribute consists of a Name, Type, and Value. 
+}
+```
+
+note that each of these fields except for 'body' can be null if no data was found, usually due to a parsing error
+for further details on body contents: [see below]
+
+## Dead Letter Archive Body contents
+
+the body attribute should be a JSON string containing an event bridge event
+
+Note that
+
+- the body attribute *can* come nested, such that you will have to de-nest a series of body attributes to get to the heart of your message
+- the word body can be interchanged with Body (capitalized)
+- because this message body arrived in the Dead Letter Archive because of issues in processing it, there is no strict guarantee that it is a valid json object, or conforms to expected structure. the *expected* structure follows.
+
+```ts
+{
+    version: [string | null], // versionString
+    id: [string | null], // unique ID of the triggering event
+    'detail-type': 'Step Functions Execution Status Change', // defines the below 'detail' spec
+    source: 'aws.states',
+    account: [string], // account ID
+    time: [string], // Zulu timestamp of the originating sf event
+    region: [string], //aws region
+    resources: [Array[string]], //ARNs of involved resources
+    detail: [string], //parses as Step Function Execution Status Change object, see below
+}
+```
+
+Step Function Execution Status Change (detail) [here](https://docs.aws.amazon.com/AWSJavaScriptSDK/v3/latest/Package/-aws-sdk-client-sfn/Interface/DescribeExecutionCommandOutput/):
+
+```ts
+{
+    stateMachineArn: [string], // ARN of the triggering workflow
+    name: [string], // Execution name of triggering execution
+    status: [string], // status of triggering execution
+    startDate: [int], // timestamp of
+    stopDate: [int | null], // timestamp of
+    input: [string], //parses as the cumulus message input
+    output: [string | null], //parses as the cumulus message output if execution succeeded
+    stateMachineVersionArn: [string | null], // The version ARN is a combination of state machine ARN and the version number separated by a colon (:)
+    stateMachineAliasArn: [string | null], // a combination of state machine ARN and the alias name separated by a colon (:)
+    /* note that these redrive statistics can be misleading, as they are not referring to the execution that failed if the triggering execution was sfEventSqsToDbRecords*/
```

Comment:
I want a second opininon on this. this is part of the message recieved by sfEvent, and so isn't a part of re-ingest stats of the actual DLA right? that seemed  alikely source of confusion unless it's me that's confused